src/test/utils/backofficevalidators.test.ts

```diff
@@ -1,13 +1,22 @@
 export {};
-//import journey from './../../../static/technicaldemo/journey.json';
-//import { componentsHaveUniqueIdentifiers } from '$lib/utils/backofficevalidators';
+import journey from './../../../static/demo_imaginary_autos/journey.json';
+import { componentsHaveUniqueIdentifiers, componentIdentifiersMustNotHaveAFullStop, componentIdentifiersMustNotBeNumerical } from '$lib/utils/backofficevalidators';
 
 // TODO: Rework journey.d.ts until journey registers as valid
 //       currently not using discriminated union correctly - picking wrong type
 describe('', () => {
-	test.todo('Component identifiers should be unique'); //    expect(componentsHaveUniqueIdentifiers(journey))
+	test.todo('Component identifiers should be unique'); 
+	expect(componentsHaveUniqueIdentifiers(journey))
+});
+
+describe('', () => {
+	test.todo('Component identifiers must not contain a full stop'); 
+	expect(componentIdentifiersMustNotHaveAFullStop(journey))
+});
+
+describe('', () => {
```

Comment:
I think numeric is fine, should be treated by the state management as a string "12345" but I wouldn't advise it, so happy to include this validation.